store/src/java-test/com/zimbra/cs/redolog/op/CreateMessageTest.java

```diff
@@ -12,17 +12,12 @@
 import com.zimbra.cs.mime.ParsedMessageDataSource;
 import com.zimbra.cs.redolog.RedoLogInput;
 import com.zimbra.cs.redolog.RedoLogOutput;
+import org.junit.*;
```

Comment:
Gren I went ahead and updated this.  Combined the change with a fixup commit back into the original commit.